baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -24,7 +24,8 @@ import { Transform } from 'stream';
 import { RequestType } from 'google-gax/build/src/apitypes';
 import * as protos from '../../protos/protos';
 import * as gapicConfig from './echo_client_config.json';
-
+import { operationsProtos } from 'google-gax';
+import { Metadata } from 'google-gax/build/src/grpc';
```

Comment:
Instead, let's export this type from gax directly

Reply:
Cool, I think we use more explicit types, this should be unused now, removing it.